launcher/src/backend/NodeConnection.js

```diff
@@ -998,6 +998,30 @@ export class NodeConnection {
     return app ? app.getVersion() : 'N/A';
   }
 
+  async getLargestVolumePath() {
+    return new Promise(async (resolve, reject) => {
+      try {
+        const dfOutput = await this.sshService.exec(
+          "df -h | sort -k 4 -r | tail -n +2 | head -n 1"
```

Comment:
remove "-h" to get size in bytes. otherwise the "sort" command will consider a volume of 247M larger than a volume of 136G, example:
![grafik](https://user-images.githubusercontent.com/107847185/213545646-77793d52-caec-4098-9ad2-31ec94414ee5.png)